docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,78 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The `ALTER VIEW` statement changes various auxiliary properties of a view.
+
+
+#### Rename view
```

Comment:
Capitalize  V?

Reply:
Maybe make this all caps for consistency, even

Reply:
@huaxingao@srowen 👍